apis/projectcontour/v1/httpproxy.go

```diff
@@ -294,6 +297,20 @@ type LoadBalancerPolicy struct {
 	Strategy string `json:"strategy,omitempty"`
 }
 
+// HeaderPolicy defines how headers are managed during forwarding
+type HeaderPolicy struct {
+	// ReplaceHeader defines how headers will be replaced during forwarding
+	ReplaceHeader []ReplaceHeader `json:"replaceHeader,omitempty"`
+}
+
+// ReplaceHeader defines how a header can be replaced during forwarding
```

Comment:
We need to define "replaced".

Reply:
Maybe for the restricted case we're introducing now; we only allow people to replace the `Host: header` which is assumed to be present (nobody look at HTTP/1.0), thus replace means _replace unconditionally_. I'm thinking that in the future if there needs to be a specific "replace only if present, otherwise don't touch", then later we can add `replaceIfPresent`.